bootstrap/bootstrap.sh

```diff
@@ -270,5 +270,9 @@ done <$rootToolVersions
 
 cp $rootToolVersions $bootstrapComplete
 
+if [ -e "$toolsLocalPath/FrameworkTargeting.targets" ]; then
```

Comment:
This works around a casing issue, we have a hack that changes the case of this file in init-tools.sh because the PCL projects expect the "Targets" extension.  Once we can take an updated FrameworkTargeting file (which I'm touching in this PR) we can remove the hack.